### PR TITLE
Don't hardcode storage class and add some convenience targets

### PR DESCRIPTION
**Pull request by viovanov**

### CODE REVIEW

File: bazel-scf-v2 [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/toaster/.cache/bazel/_bazel_toaster/0291e00957d43be0bd77a17ccdc76a11/execroot/scf
```
Comment (f0rmiga) [inline bazel-scf-v2]: This symlink slipped out the generated Bazel directories. We need to update the `.gitignore` to include `/bazel-*`.

File: deploy/helm/scf/assets/operations/instance_groups/database.yaml [hunk 2]
```diff
@@ -122,6 +121,9 @@
       processes:
       - name: mariadb_ctrl
         executable: /bin/bash
+        limits:
```
Comment (f0rmiga) [inline deploy/helm/scf/assets/operations/instance_groups/database.yaml]: Any particular reason to move these? I'd prefer having the `args` right after the `executable` as they together make the full command.

File: deploy/helm/scf/values.yaml [hunk 3]
```diff
@@ -1 +1,2 @@
 system_domain: ~
+asd: asd
```
Comment (f0rmiga) [inline deploy/helm/scf/values.yaml]: Left-over?

File: deploy/helm/scf/assets/operations/instance_groups/database.yaml (hunk 2 above)
Comment (viovanov) [reply deploy/helm/scf/assets/operations/instance_groups/database.yaml]: it's hard for me to read where the keys belong if they're after the long script
can't enforce something like this .. but I'll put them before `executable`

File: deploy/helm/scf/values.yaml (hunk 3 above)
Comment (viovanov) [reply deploy/helm/scf/values.yaml]: :D

File: deploy/helm/scf/assets/operations/instance_groups/database.yaml (hunk 2 above)
Comment (f0rmiga) [reply deploy/helm/scf/assets/operations/instance_groups/database.yaml]: I see. This works too.

**Pull request merged**